test/putTransferSpec.js

```diff
@@ -1018,103 +1016,7 @@ describe('PUT /transfers/:id', function () {
         .end()
     })
 
-  /* Subscriptions */
-  it('should trigger subscriptions', function * () {
```

Comment:
Is this not sufficient: https://github.com/interledger/five-bells-ledger/blob/refactor_notifications/test/accountSpec.js#L603 ?